internal/rbd/replicationcontrollerserver.go

```diff
@@ -300,23 +300,56 @@ func getOperationName(poolName string, optName operation) string {
 // createDummyImage creates a dummy image as a workaround for the rbd
 // scheduling problem.
 func createDummyImage(ctx context.Context, rbdVol *rbdVolume) error {
+	var err error
+	var imgName string
 	optName := getOperationName(rbdVol.Pool, dummyImageCreated)
 	if _, ok := operationLock.Load(optName); !ok {
```

Comment:
To prevent parallel dummy image creation or size update, should we just add the:
```
	dummyImageOpsLock.Lock()
	defer dummyImageOpsLock.Unlock()
```
here as well?

Previously image exists was ignored, and so a race to create would not touch the image. Now as it is being recreated (even when size is the same) it is potentially possible that the first call is now attempting the tickle of the schedule when the subsequent call is deleting the image.